### PR TITLE
What Is: What Are Conferences And Workshops

### LINKED ISSUE #1598: Cleanup "WhatIs" files for all topics and updated EBdocs
Create/edit/delete "WhatIs" files for all topics and updated EBdocs as listed below

1. [x] Make list of all the WhatIs files for all categories and topics? Make a note of what is native, pdf or missing? (@rinkug )
2. [x] Assign folks to missing what-is files
3. [x] Assign folks to update pdf files
4. [x] Unpublish obsolete WhatIs files
5. [ ] Assign folks to review existing files for new content. Excel sheet can be found [HERE](https://docs.google.com/spreadsheets/d/1Bp7LHdhwnfncdp8l8o5QxkpWpG4i0gcK/edit#gid=1890549563).
6. [x] Remove "pinned:yes" from other non-WhatIs files (@rinkug PR #1599 ) 
7. [x] EB docs: Add all new topics (inclusivity, rse etc)  to EB docs (PR #1596)
9. [x] EB docs: Make section on how to create WhatIs files and why they are needed (PR #1596)
10. [ ] Decide: Where do WhatIs files fit in the github directory hierarchy?

Related to #560

### PR DESCRIPTION
**Pull request by rinkug**
Author: @rinkug 
EB Member: @markcmiller86 
Partly Resolves #1598 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [ ] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [x] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [x] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

### CODE REVIEW

**Comment by rinkug** [comment]
The article can do with a better titles as long as the starting format is "What are conferences and workshop......"

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#### Contributed by  [Rinku Gupta](https://github.com/rinkug)
+#### Publication date: April 20, 2023
+
+<!--- deck start --->
+Empowering researchers in scientifivc computing with knowledge-sharing, collaboration, and networking opportunities through conferences and workshops.
+<!--- deck end --->
+
+<!--- body start --->
+
+Conferences and workshops play a pivotal role in advancing scientific research in the field of scientific computing. These events offer researchers a valuable platform to showcase their latest findings, discuss and exchange ideas with other experts, and cultivate collaborations that drive innovation in their field. Beyond knowledge-sharing and collaboration, attending conferences and workshops enables researchers to remain up-to-date on the latest trends and advancements in their field, enabling them to refine their research approach and stay ahead of the curve.
```
Comment (markcmiller86) [inline Articles/WhatIs/WhatAreConferencesAndWorkshops.md:11]: Do they *advance* the research itself or primarily serve as a forum for *communicating* the research? I think the latter.

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Conferences and workshops provide researchers with a unique opportunity to present their work to an audience of peers, fostering critical discussions that encourage the exchange of ideas and feedback. Through these platforms, researchers receive constructive criticism that enables them to fine-tune their ideas and approaches, resulting in the development of new insights, techniques, and approaches that can further advance their research agenda. Scientific computing software stacks are built for various applications from different domains. Conferences and workshops offer researchers a chance to connect with experts from other domains and fields, fostering interdisciplinary collaboration and knowledge-sharing. These events also provide excellent opportunities for industry, academia, national laboratories, and other research organizations to connect and exchange notes. Several of these factors result in awareness of new funding and grant opportunities. With both formal and informal settings, conferences/workshops are considered good environments to build deeper, meaningful relationships with one's colleagues and advance in the field.
```
Comment (markcmiller86) [inline Articles/WhatIs/WhatAreConferencesAndWorkshops.md:13]: * platforms --> foums
* The sentence on research stacks seems misplaced...maybe remove? 
* Last few sentences seem a repeat of end of paragraph 1 with exception of awareness of *funding* too. I think most of this paragraph could be collapsed into single new sentence in paragraph 1.

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Networking and career advancement opportunities are other significant outcomes of attending conferences and workshops. Employers in the scientific computing field often use these events to recruit talented interns/employees who possess the drive, enthusiasm, skills, knowledge, or experience to help them achieve their goals. These venues provide an opportunity for all these organizations to promote and highlight their mission, values, and the projects they are currently working on. For younger researchers, conferences/workshops provide a platform to showcase their work, whether it's through presenting a poster, giving a talk, or participating in a panel. This can help them gain recognition for their research and raise their profile in their field.
```
Comment (markcmiller86) [inline Articles/WhatIs/WhatAreConferencesAndWorkshops.md:15]: I guess I see a lot of repeat of some of the same basic ideas from previous paragraphs here. 

I think all of these points are great points to make.

I just think there is a far less wordy way of making them. And, it was my understanding that part of the goal of this particular style (e.g. what-is docs) of documents was to be short and sweet.

**Review by markcmiller86 — CHANGES_REQUESTED** [review]
I think all great points are made here but it just seems a bit repetitive in that some of the same points are made with different wording in each paragraph.

**Review by pagrubel — CHANGES_REQUESTED** [review]
@rinkug As an older but newer researcher, I think you should try to get rid of the term younger and replace it with new researcher or somethink like a researcher new to the field, or emerging researcher (so no ageism)

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md (hunk 1 above)
Comment (pagrubel) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md:11]: @rinkug As an older but newer researcher, I think you should try to get rid of the term younger and replace it with new researcher or somethink like a researcher new to the field, or emerging researcher (so no ageism)

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md (hunk 3 above)
Comment (markcmiller86) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md:15]: I can try to take a whack later today if you want me too.

Comment (rinkug) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md:15]: Pleasee do!!!

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md
[the remark this answers is not in the document]
Comment (rinkug) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md:1]: Sounds good

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md (hunk 2 above)
Comment (rinkug) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md]: Sounds good

File: Articles/WhatIs/WhatAreConferencesAndWorkshops.md (hunk 1 above)
Comment (rinkug) [reply Articles/WhatIs/WhatAreConferencesAndWorkshops.md]: sounds good!

**Review by rinkug — COMMENTED** [review]
Thanks for feedback

**Review by rinkug — COMMENTED** [review]
Thanks for review

**Comment by rinkug** [comment]
@markcmiller86 : corrected grammar and added a sentence or two. Ready to publish from my side

**Review by markcmiller86 — APPROVED** [review]

**Pull request merged**